deployment/desktop/how-to-propose-a-3rd-party-application-for-inclusion-in-fedora.md

```diff
@@ -0,0 +1,9 @@
+[Back to index](how-to-package-a-desktop-application.md) 
+
+###Getting it into Fedora Workstation
+You now have your repository created and ready for users to connect to it. The final step is getting your repository added to Fedora Workstation.
+To start the process for this you can file a [Fedora ticket](https://bugzilla.redhat.com/enter_bug.cgi?product=Fedora) against the Fedora-workstation-repositories component. The information needed is a decription of your application. We also need the URL where the repository is located.
+
+This will allow the Fedora Workstation Working group to review your application and ask any needed follow-up questions. For more in depth-information about the process and rules for including 3rd party software in Fedora Workstation you can see the [Third Party Software for Fedora](https://fedoraproject.org/wiki/Workstation/Third_party_software_proposal) page.
+
+[Back to index](how-to-package-a-desktop-application.md) 
```

Comment:
These `back to index` are unnecessary, because there is a link in left panel pointing to that page. This link(ending with `.md`) would also most probably not work.